test/powershell/Modules/Microsoft.PowerShell.Management/PSDrive.Tests.ps1

```diff
@@ -1,5 +1,23 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
+
+Describe "Tests with elevated permissions" -Tag "CI","RequireAdminOnWindows" {
+    Context "Validate PSDrive Cmdlets" {
+        It "Create a new persistent PSDrive targetting remote Windows share." -Skip:(-not $IsWindows) {
+            try {
+                #Arrange
+                $RemoteShare = "\\$env:COMPUTERNAME\$($env:SystemDrive.replace(':','$\'))"
+
+                #Act and Assert
```

Comment:
We could remove the useless comment and comment in line 8 but make title of the test more informative.

Reply:
Can you suggest me a title ?
Is it for only `it` block ?

Reply:
I see that the PSDrive.Tests.ps1 file is only for alias provider tests.
I think we should move the test to new file New-PSDrive.Tests.ps1.
It would be nice if you added some new tests for New-PSDrive and renamed the test to address the current bug fix with slashes.

Reply:
sure

Reply:
@iSazonov I have added new test script, I can see many are already covered in PSDrive.Tests.ps1. Added tests for scenarios with `-Persist` switch in new file. Let me know if we have more scenarios to cover in new file.

Reply:
Thanks for your contribution! 
I can not ask you to add more tests :-)